Wallix/wallix-bastion/tests/test_new_process_event.json

```
@@ -0,0 +1,49 @@
+{
+  "input": {
+    "message": "2025-04-23T16:24:52+02:00 test01 rdpproxy 32019 - - [RDP Session] session_id=\"19662b5f089766d40050000400000\" client_ip=\"1.1.1.1\" target_ip=\"17.1.11.11\" user=\"test@test.fr\" device=\"test_device\" service=\"RDP\" account=\"test@test.fr\" type=\"NEW_PROCESS\" command_line=\"\\\"C:\\\\Windows\\\\system32\\\\test_script.exe\\\" -ServerName:serverTest\"",
+    "sekoiaio": {
+      "intake": {
+        "dialect": "WALLIX Bastion",
+        "dialect_uuid": "6b8cb346-6605-4240-ac15-3828627ba899"
+      }
+    }
+  },
+  "expected": {
+    "message": "2025-04-23T16:24:52+02:00 test01 rdpproxy 32019 - - [RDP Session] session_id=\"19662b5f089766d40050000400000\" client_ip=\"1.1.1.1\" target_ip=\"17.1.11.11\" user=\"test@test.fr\" device=\"test_device\" service=\"RDP\" account=\"test@test.fr\" type=\"NEW_PROCESS\" command_line=\"\\\"C:\\\\Windows\\\\system32\\\\test_script.exe\\\" -ServerName:serverTest\"",
+    "event": {
+      "action": "NEW_PROCESS"
+    },
+    "destination": {
+      "address": "17.1.11.11",
+      "ip": "17.1.11.11"
+    },
+    "host": {
+      "name": "test_device"
+    },
+    "process": {
+      "command_line": "\\\"C:\\\\Windows\\\\system32\\\\test_script.exe\\\" -ServerName:serverTest"
```

Can you remove the double-backslahes?

```suggestion
      "command_line": "\\\"C:\\Windows\\system32\\test_script.exe\\\" -ServerName:serverTest"
```

see [this issue](https://github.com/SekoiaLab/integration/issues/550) and [this PR](https://github.com/SEKOIA-IO/intake-formats/pull/1605)